src/core/less/theme/colors.less

```diff
@@ -0,0 +1,145 @@
+// Custom Colors
+// =============
+
+@white: #ffffff;
+@black: #4b4e4f;
+@blue: #7D7D7D;
+@light-grey: #78909c;
+@mid-grey: #546e7a;
+@dark-grey: #37474f;
+@red: #f26c4f;
+@green: #82ca9c;
+
```

Comment:
Suggest dropping the arbitrary list of 'custom' colours in this non-theme and replacing the subsequent named colours in this file with hex codes?